src/components/PupilComponents/QuizMCQMultiAnswer/QuizMCQMultiAnswer.tsx

```diff
@@ -22,18 +22,13 @@ import {
   StemTextObject,
 } from "@/node-lib/curriculum-api-2023/shared.schema";
 import { useQuizEngineContext } from "@/components/PupilComponents/QuizEngineProvider";
-import { useInitialChange } from "@/components/PupilComponents/QuizUtils/useInitialChange";
 import { getSizes } from "@/components/SharedComponents/CMSImage/getSizes";
 
 export type QuizMCQMultiAnswerProps = {
-  onInitialChange?: () => void;
-  onChange?: () => void;
+  onChange: () => void;
```

Comment:
I ditched the `onInitialChange` handler since I needed to be able to update the question "mode" whenever a change was made to the question answer. It

Reply:
it ... ?

Reply:
Looks like we encountered an unexpected EOL in my stream of consciousness.